### PR TITLE
Updating edit comment styling

### PR DESCRIPTION
**Pull request by mdbenjam**
The current styling of our edit comment text box is ugly.
screen shot 2017-04-19 at 5 30 58 pm

Connects: #1283 

**Test Plan**
- [x] Add a comment and make sure the new comment box looks like the mock.

### CODE REVIEW

File: client/app/components/EditComment.jsx [hunk 1]
```diff
@@ -58,16 +52,14 @@ export default class EditComment extends BaseForm {
...
-          className="comment-container">
-          <TextareaField
-            id={this.props.id}
-            label="Edit Comment"
-            name="editComment"
-            onChange={this.handleFieldChange('commentForm', 'editComment')}
-            {...this.state.commentForm.editComment}
-          />
-        </div>
+        <textarea
```
Comment (mdbenjam) [inline client/app/components/EditComment.jsx]: @joofsh What do you think of using textarea instead of our component, when we want a very specific styling?

Comment (NickHeiner) [inline client/app/components/EditComment.jsx]: Let's chat in person about this.

Comment (NickHeiner) [inline client/app/components/EditComment.jsx]: Sounds good to me.

**Comment by NickHeiner** [comment]
Maybe I've got something screwed up locally, but this seems like it's reverting earlier fixes (which I think were dbee510435895f211ce71185f72d559618fcb006):

image

**Comment by NickHeiner** [comment]
Aside from that, LGTM.

**Pull request merged**